### ISSUE TITLE
Integrating with `hal`

### ISSUE THREAD

**Issue by elsirion** [MutinyWallet/bitcoin-waila#1]
I love the idea of the project! There is a similar debug tool already, [`hal`](https://github.com/stevenroose/hal/) by @stevenroose, this might be a nice thing to integrate there instead of having yet another tool one needs to remember to install. It could be `hal waila <string>`.

**Comment by benthecarman** [comment]
good idea, the intention of this project is more to be a library than command line tool, would definitely make sense to add to Hal though.
